src/components/sdk/MeetingControls/AudioInputVFControl.tsx

```diff
@@ -0,0 +1,146 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { useEffect, useState, ReactNode } from 'react';
+import { Device, AudioTransformDevice, VoiceFocusTransformDevice } from 'amazon-chime-sdk-js';
+
+import { ControlBarButton } from '../../ui/ControlBar/ControlBarButton';
+import { DeviceConfig } from '../../../types';
+import { isOptionActive, audioInputSelectionToDevice } from '../../../utils/device-utils';
+import { Microphone } from '../../ui/icons';
+import { PopOverItem } from '../../ui/PopOver/PopOverItem';
+import { PopOverSeparator } from '../../..';
+import { Spinner } from '../../ui/icons';
+import { useMeetingManager } from '../../../providers/MeetingProvider';
+import { useAudioInputs } from '../../../providers/DevicesProvider';
+import { useVoiceFocus } from '../../../providers/VoiceFocusProvider';
+import { useToggleLocalMute } from '../../../hooks/sdk/useToggleLocalMute';
+
+interface Props {
+  /** The label that will be shown when microphone is muted, it defaults to `Mute`. */
+  muteLabel?: string;
+  /** The label that will be shown when microphone is unmuted, it defaults to `Unmute`. */
+  unmuteLabel?: string;
+  /** Title attribute for the icon when muted, it defaults to `Muted microphone` in <Microphone />. */
+  mutedIconTitle?: string;
+  /** Title attribute for the icon when unmuted, it defaults to `Microphone` in <Microphone />. */
+  unmutedIconTitle?: string;
+  /** The label that will be shown when the current input audio is voice focus device, it defaults to `Voice focus is enabled`. */
+  voiceFocusOnLabel?: string;
+  /** The label that will be shown when the current input audio is not voice focus device, it defaults to `Enable voice focus`. */
```

Comment:
Always capitalize "Voice Focus", and whenever used in human-readable situations (e.g., labels, CHANGELOG), use the full string "Amazon Voice Focus".

Reply:
Thanks for the reminder! Changed it to "Amazon Voice Focus" throughout.